hutch_python/epics_arch.py

```diff
@@ -182,16 +351,32 @@ def create_file(exp_name, path):
     path : str
         Directory where to create the epicsArch file.
     """
-    data_list = get_questionnaire_data(exp_name)
-    if not os.path.exists(path):
-        raise OSError('Invalid path: %s' % path)
-    exp_name = str(exp_name)
-    file_path = ''.join((path, 'epicsArch_', exp_name, '.txt'))
+    qs_data = get_questionnaire_data(exp_name)
 
-    logger.info('Creating epicsArch file for experiment: %s', exp_name)
+    logger.debug("UpdateFile: qs_data:\n" + str(qs_data))
 
+    logger.debug("\nPath: " + str(path))
+    af_path = str(path) + "epicsArch_" + str(exp_name) + ".txt"
+    logger.debug("\nAF Path: " + str(af_path))
+    exp_name = str(exp_name)
```

Comment:
nitpick: you seem to be repeatedly doing `str(exp_name)` both before and after this line. You can just do this once earlier and not need to repeatedly recast as `str` later. You can similarly do this for `path`.